frontend/cypress/integration/admin/pool-candidate.spec.js

```diff
@@ -20,93 +20,118 @@ describe("Pool Candidates", () => {
       aliasQuery(req, "GetPoolCandidatesPaginated");
 
       aliasMutation(req, "UpdatePoolCandidateStatus");
+
+      // Only used if FEATURE_APPLICANTSEARCH is false
+      aliasQuery(req, "getUpdatePoolCandidateData");
+      aliasMutation(req, "updatePoolCandidate");
     });
 
     loginAndGoToPoolsPage();
   });
 
-  it("should update pool candidate status", () => {
-    cy.wait("@gqlgetPoolsQuery");
+  if (Cypress.env("FEATURE_APPLICANTSEARCH")) {
+    it("should update pool candidate status", () => {
```

Comment:
Maybe just make it more clear in the test output which ones ran/passed/failed?

```suggestion
    it("should update pool candidate status (FEATURE_APPLICANTSEARCH:on)", () => {
```